### PR TITLE
Feature branch : Todo App

### PR DESCRIPTION
**Pull request by ehmaddd**
In this PR, I have applied multiple features related to Todo List and all are based on React.
The glimpse of features is : 
- You can add a task
- You can delete a task
- You can mark a task as done
- All actions are based on React DOM

### CODE REVIEW

File: src/components/InputTodo.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    <>
+      <form onSubmit={handleSubmit} className="form-container">
+        <input
+          type="text"
+          placeholder="Add Todo..."
+          value={title}
+          onChange={handleChange}
+          className="input-text"
+        />
+        <button type="button" className="input-submit">Submit</button>
```
Comment (abeltsew) [inline src/components/InputTodo.js:32]: Changing the button type to "submit", will insure that the button will work in addition to the Enter press

File: src/App.js
Comment (abeltsew) [inline src/App.js:1]: please remove unused files"App.js" to avoid confusion.

**Review by abeltsew — APPROVED** [review]
Hi, @ehmaddd 👋,

##

### Highlights💡
- Good commit message.✔️ 
- Good PR Summary.✔️
- App functions as it should.✔️

##

# STATUS: APPROVED 🕺🥳 

Congratulations🍾🥳!! Your project is ready for presentation!  There is nothing else to say other than... it's time to merge it :shipit: 




##

###  **Optional Suggestions:**

- [x]  Remove Unused Files
- [x]  change button to be "submit" type to trigger form submission

File: src/components/InputTodo.js (hunk 1 above)
Comment (ehmaddd) [reply src/components/InputTodo.js:32]: That's a great idea. Let me apply it immediately

File: src/App.js
Comment (ehmaddd) [reply src/App.js:1]: Nice suggestion.

**Pull request merged**